cockatrice/src/dlg_settings.cpp

```diff
@@ -246,6 +249,7 @@ void GeneralSettingsPage::retranslateUi()
     highQualityURLLabel.setText(tr("Custom Card Download URL:"));
     highQualityURLLinkLabel.setText(QString("<a href='%1'>%2</a>").arg(LINKING_FAQ_URL).arg(tr("Linking FAQ")));
     clearDownloadedPicsButton.setText(tr("Reset/Clear Downloaded Pictures"));
+    updateNotificationCheckBox.setText(tr("Notify when client updates are available"));
```

Comment:
I think the most accurate wording here would be `Allow servers to inform you about unsupported features`.

Honestly I don't think it should be configurable at all - consider the case of reveal bottom card, where if you lack it, someone else can do game actions that you can't see - we shouldn't allow people to miss this information.

And if they're missing required and not optional features, then we have to tell them no matter what the setting.